Kitodo/src/main/java/de/sub/goobi/forms/ProzessverwaltungForm.java

```diff
@@ -2164,7 +2118,7 @@ public void setProperty(Property property) {
     }
 
     /**
-     * Set list of process properties.
+     * Set list of process' properties.
```

Comment:
Is adding of this apostrophe `'` really wanted?

Reply:
According to English grammar? Yes ;)